### PR TITLE
Update team.md

### PR DESCRIPTION
**Pull request by fangninghe**

### CODE REVIEW

File: pages/team.md [hunk 1]
```diff
@@ -1,38 +1,42 @@
...
-* [FanisGk](FanisGk.md)
-* [mepda](mepda.md)
-* [prmurphy](prmurphy.md)
-* [hikaruit15](hikaruit15.md)
-* [waryhermit](waryhermit.md)
-* [IshanSoni](IshanSoni.md)
-* [hannahgdubin](hannahgdubin.md)
+|**Username**|**Join Date**|
+|------------|-------------|
+|[waryhermit](waryhermit.md)(Internlead)|20161213|
```
Comment (hikaruit15) [inline pages/team.md]: We should have a `space` between name and title Internlead.

Comment (hikaruit15) [inline pages/team.md]: Do the same for other interns that have title.

**Comment by fangninghe** [comment]
@hikaruit15 , fixed the space before internlead

**Comment by hikaruit15** [comment]
hi @fangninghe, one more thing: `intern lead` instead of `Internlead`

**Comment by fangninghe** [comment]
@hikaruit15 , cool and done.

**Comment by dogi** [comment]
@fangninghe got this over gitter ;)
> Che-Ting Lin @ctl74100 17:50
Hi dogi,
i'm one of the virtual intern from last year
join date: 6/3 leave date:8/11

**Comment by fangninghe** [comment]
@dogi , updated @ctl74100

**Comment by fangninghe** [comment]
@waryhermit ,thanks, it looks better

**Comment by fangninghe** [comment]
@dogi , it is really my pleasure and I am looking forward to joining you guys!

**Pull request merged**